pytest_html/plugin.py

```diff
@@ -320,14 +330,20 @@ def generate_summary_item(self):
             unicode_doc = unicode_doc.decode('utf-8')
         return unicode_doc
 
-    def _save_report(self, report_content):
-        if not os.path.exists(os.path.dirname(self.logfile)):
-            os.makedirs(os.path.dirname(self.logfile))
+    def _save_report(self, report_content, self_contained_html):
+        dir_name = os.path.dirname(self.logfile)
+        if not os.path.exists(dir_name):
+            os.makedirs(dir_name)
         logfile = open(self.logfile, 'w', encoding='utf-8')
-
         logfile.write(report_content)
         logfile.close()
 
+        if not self_contained_html:
+            style_path = dir_name + '/style.css'
```

Comment:
This won't work on Windows where the path separator is `\`. Use [`os.path.join`](https://docs.python.org/2/library/os.path.html#os.path.join)